Readme.md

```diff
@@ -18,7 +22,8 @@ MegaMol succeeds ​MolCloud, which has been developed at the University of Stut
 5. Run `make install` to create your MegaMol installation
 6. Test Megamol with
 
-        ./megamol.sh -p ..\examples\testspheres.lua
+        $./megamol.sh -p ../examples/testspheres.lua
```

Comment:
I find the $ symbol in the beginning misleading. At first I thought it was part of the command. Maybe add a space, such that it reads `$ ./megamol` or even `LINUX$ ./megamol` as done in the docs/manual.md file.

Reply:
Jupp, I changed it consitantly in commit 5a94a3e.